jasmine/tests/shard.test.js

```diff
@@ -0,0 +1,26 @@
+describe('test sharding with explicit isolation failures', () => {
```

Comment:
which "failures"? 🤔

Reply:
That's how I understood it when I read the code, but the description here mentions "failures" and I'm not sure what those failures are..?

Reply:
yeah, i am not sure either. open to suggestions.

Reply:
changed it to "assert sharding runs specs(it blocks) in the expected order and partitions equally over the available shards"